deploy/src/main/resources/quick-start/README.md

```diff
@@ -0,0 +1,214 @@
+
+3. **Set the service labels**
+
+	For each of the services to scale its docker-compose template must be updated to include labels such as the following example:
+
+	```
+   		labels:
+      		- autoscale.metric=rabbitmq
+      		- autoscale.scalingtarget=autoscale-monitor-in
+      		- autoscale.scalingprofile=default
+      		- autoscale.maxinstances=4
+      		- autoscale.mininstances=1
+      		- autoscale.interval=30
+      		- autoscale.backoff=10
+	```
+
+	The `autoscale.metric` should always be `rabbitmq` for this container. Set the `autoscale.interval` to something sensible given your service workload. If you set this to a very small value (very frequent) then you will get very unstable scaling - the number of instances will rapidly increase and decrease seemingly with no cause. if you set this very large (infrequent) then the scaling will lag behind the demand significantly. Thus, containers which rapidly consume messages can have a lower interval. For containers that consume messages very slowly (whereby the average is tens of minutes or hours) this algorithm may not be suitable for your workloads and you may wish to consider writing a `workload` module more suited to your needs and make a container with it in.
+
+	The `autoscale.scalingtarget` should be the RabbitMQ queue name to monitor. Set `autoscale.maxinstances` and `autoscale.mininstances` as your quality of service and resources dictate. The `autoscale.backoff` is the number of intervals to skip monitoring after a scale up or down command is issued. This prevents unusual values being considered when the system is in an unstable state.
+
+	Deploy/redeploy the services and the autoscale container. After one or two minutes the autoscale container should find the services and start monitoring.
+
+	##### Example
+
+	The Job Service microservices already have these labels specified. You can see the labels set under some of the components, for example with worker-langdetect.
+
+	```
+    	labels:
+      	- autoscale.metric=rabbitmq
+      	- autoscale.scalingtarget=languageidentification-in
+      	- autoscale.scalingprofile=default
+      	- autoscale.maxinstances=4
+      	- autoscale.mininstances=1
+      	- autoscale.interval=30
+      	- autoscale.backoff=10
+	```
+
+
+4. **Deploy the services**
+ 
+ 	Navigate to the folder where you have downloaded the files to and issue the following command to deploy the services through Docker Stack. Each service provided in the compose file will be deployed to the same stack. The Autoscaler needs to be on the same stack in order to monitor other services. In the command below the stack name is `autoscalerdemo`
+
+	<table>
+      <tr>
+        <td><b>Docker Stack</b></td>
+        <td>docker stack deploy --compose-file=docker-compose.yml autoscalerdemo</td>
+      </tr>
+    </table>
+
+	##### Example
+
+	Running this command with the docker-compose.yml file supplied will bring up the Job Service as well as the Autoscaler Service.
```

Comment:
Can we say something more to say that we are brining up the autoscalerdemo which consists of the Job service microservices, workers which can be scaled and the Autoscaler.  ( something like this ).